pkgs/tools/archivers/wiseunpacker/default.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchFromGitHub
+, buildDotnetModule
+, dotnetCorePackages
+, lib
+}:
+let
+  version = "1.3.3";
+  pname = "WiseUnpacker";
+in
+buildDotnetModule rec {
+  inherit version pname;
+
+  src = fetchFromGitHub {
+    owner = "mnadareski";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-APbfo2D/p733AwNNByu5MvC9LA8WW4mAzq6t2w/YNrs=";
+  };
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+
+  dotnetInstallFlags = [ "-f net8.0" ];
```

Comment:
If we did something like `dotnetFlags = "-p:TargetFrameworks=net8.0";`, would it cut down on the dependencies?

Reply:
When I add this I get 
```
Executing dotnetConfigureHook
  Determining projects to restore...
/nix/store/1nmjsx4wkr7braiz5spq84b1ibvvz2jb-dotnet-sdk-8.0.303/sdk/8.0.303/NuGet.targets(169,5): error : Invalid restore input. Invalid target framework 'unsupported'. Input files: /build/source/Test/Test.csproj. [/build/source/Test/Test.csproj]
```

Reply:
I just tried it, and I was able to fetch-deps without errors.  It failed during publish without `-f`, but I found that you can do: `-p:TargetFramework=net8.0` (singular), and it makes publish happy:

```
diff --git a/pkgs/by-name/wi/wiseunpacker/package.nix b/pkgs/by-name/wi/wiseunpacker/package.nix
index ef0584fe3463..f5d42fa68f9f 100644
--- a/pkgs/by-name/wi/wiseunpacker/package.nix
+++ b/pkgs/by-name/wi/wiseunpacker/package.nix
@@ -21,7 +21,7 @@ buildDotnetModule rec {
   dotnet-sdk = dotnetCorePackages.sdk_8_0;
   dotnet-runtime = dotnetCorePackages.runtime_8_0;
 
-  dotnetInstallFlags = [ "-f net8.0" ];
+  dotnetFlags = [ "-p:TargetFramework=net8.0" ];
 
   # Rename to something sensible
   postFixup = ''
```

This also removes almost all of the dependencies, including the one with the uri escaped paths.

Reply:
Ah ok sounds good, sorry I'm not that familiar with C#, but this change works on my end too.